tfhe/c_api_tests/test_high_level_integers.c

```diff
@@ -371,9 +371,8 @@ int uint8_public_key(const ClientKey *client_key, const PublicKey *public_key) {
 
 int uint8_safe_serialization(const ClientKey *client_key, const ServerKey *server_key) {
   int ok;
-  CompactFheUint8 *lhs = NULL;
-  CompactFheUint8 *deserialized_lhs = NULL;
```

Comment:
the notion of compact ciphertext disappears in C ? could it be a thin wrapper around the heterogeneous list perhaps ? Not sure if it's worth it

Reply:
It disapears totally from the Lib